apps/web/components/Logo.tsx

```diff
@@ -1,14 +1,19 @@
+import Image from "next/image";
+
 export default function Logo({ small, icon }: { small?: boolean; icon?: boolean }) {
   return (
     <h1 className="inline">
       <strong>
         {icon ? (
-          <img className="mx-auto w-9" alt="Cal" title="Cal" src="/cal-com-icon-white.svg" />
+          <Image width={36} height={36} alt="Cal" title="Cal" src="/cal-com-icon-white.svg" layout="fixed" />
```

Comment:
Why the className here couldn't be the same?

Reply:
Unfortunately, the Image component from Next disregards any Tailwind class applied to it, so any size was converted to `width` and `height` and any other class was needed, a wrapper with those classes needs to be created.

Reply:
This falls into "over optimization" territory IMO. We don't need `next-image` for these SVGs and icons. Let's just use `img` and suppress the eslint warning with `eslint-disable @next/next/no-img-element` So make it explicit that we don't have to optimize these on purpose.